lms/templates/courseware/proctored-exam-status.underscore

```diff
@@ -8,7 +8,7 @@
     %>
     <div class='exam-text'>
         <%= interpolate_text('You are taking "{exam_link}" as a {exam_type} exam. The timer on the right shows the time remaining in the exam.', {exam_link: "<a href='" + exam_url_path + "'>"+gtLtEscape(exam_display_name)+"</a>", exam_type: (!_.isUndefined(arguments[0].exam_type)) ? exam_type : gettext('timed')}) %>
-        <%- gettext('To receive credit on a problem, you must click "Check" or "Final Check" on it before you select "End My Exam".') %>
+        <%- gettext('To receive credit for problems, you must click "Submit" on it before you select "End My Exam".') %>
```

Comment:
@staubina Thanks for the quick response also! Just missed the "on it" -> "for each problem" now that the first part of the sentence is plural.
"...you must click "Submit" for each problem before you select..."